HSLdevcom/helmet-ui
# helmet-ui 175: Add header link to documentation

How to test:

Pull branch locally, start the app, click the book icon on the header and it should open up a new browser tab/window with the helmet-ui documentation.
